schemas/v1.0/schema.yaml

```
@@ -1,5 +1,101 @@
-$id: https://spec.openapis.org/overlay/1.0/schema/WORK-IN-PROGRESS
+$id: https://spec.openapis.org/overlay/1.0/schema/2024-10-17
 $schema: https://json-schema.org/draft/2020-12/schema
-description: The description of Overlay v1.0.x documents
-
+description: The description of OpenAPI Initiative Overlay v1.0.0 documents
+  without schema validation, as defined by
+  https://spec.openapis.org/overlay/v1.0.0
```

```suggestion
description: The description of Overlay v1.0.x documents
```
Following https://github.com/OAI/OpenAPI-Specification/blob/main/schemas/v3.0/schema.yaml